cypress.json

```diff
@@ -3,7 +3,7 @@
     "TAGS": "not @ignore and @test"
   },
   "testFiles": "**/*.feature",
-  "defaultCommandTimeout": 30000,
+  "defaultCommandTimeout": 60000,
```

Comment:
Kanskje bedre å øke grensa bare hvor vi vet det er behov? https://docs.cypress.io/guides/core-concepts/retry-ability#Increase-time-to-retry

Hivs ikke vil vel alle tester som feiler ta 3sek ekstra hver

Reply:
Endret det da det er litt for mange steder det av og til går i timeout pga treghet i systemet. Mener det ikke er samme sted hver gang, men kan prøve å holde et øye med det.